api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/audit/EventAuditor.java

```diff
@@ -58,17 +57,17 @@ public void auditUpdatedClaim(Claim claim, List<UpdatableClaimantField> updatedF
     /**
      * Audit a new card event given a card response.
      *
-     * @param claimId      The claim id
-     * @param cardResponse The card response which must not be null
+     * @param claimId      The claim id which must not be null
+     * @param cardAccountId The card account id which must not be null
      */
-    public void auditNewCard(UUID claimId, CardResponse cardResponse) {
-        if (claimId == null || cardResponse == null) {
-            log.warn("Unable to audit new card event with claimId: {} and cardResponse: {}. Both fields must not be null", claimId, cardResponse);
+    public void auditNewCard(UUID claimId, String cardAccountId) {
+        if (claimId == null || cardAccountId == null) {
```

Comment:
Is it worth being a bit more strict on the cardAccountId and using `StringUtils.isBlank()`. I guess the question is whether or not a cardAccountId which is a blank string is acceptable or not.